release-notes/2023058-ovn-gw-node.md

```diff
@@ -0,0 +1,2 @@
+<!-- markdownlint-disable MD041 -->
```

Comment:
Also please rename the file to `20230508-ovn-gw-node.md`

Reply:
done.